perf_bench_test.go

```diff
@@ -705,19 +706,11 @@ func bindProcessToCPU(pid string, processors ...int) error {
 
 func getCoresExceptProfiled(cpuInfo profile.CPUInfo, cpus []int) ([]int, error) {
 	var (
-		sibling int = -1
 		subList []int
 	)
 
-	if *profileCPUID > -1 {
-		cpu, err := cpuInfo.GetSibling(*profileCPUID)
-		if err != nil {
-			return nil, err
-		}
-		sibling = cpu
-	}
 	for _, cpu := range cpus {
-		if *profileCPUID != cpu && sibling != cpu {
+		if *profileCPUID != cpu {
```

Comment:
you need to allow VMs to be scheduled on the profile core. Also, one VM should always be on that core.

Reply:
Done